debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImageProcess.java

```diff
@@ -578,24 +566,25 @@ public J9DDRBaseImageThread getThread(long id) {
 		Map<Long, Object> dtfjThreadMap = getThreadMap();
 		return (J9DDRBaseImageThread)dtfjThreadMap.get(id);
 	}
-	
+
 	private int resolveGenericSignal(int num) {
-		if ((num & J9PORT_SIG_FLAG_SIGQUIT) != 0) 	return 3;
-		if ((num & J9PORT_SIG_FLAG_SIGILL) != 0) 	return 4;
-		if ((num & J9PORT_SIG_FLAG_SIGTRAP) != 0) 	return 5;
-		if ((num & J9PORT_SIG_FLAG_SIGABRT) != 0) 	return 6;
-		if ((num & J9PORT_SIG_FLAG_SIGFPE) != 0) {
-			if (num == J9PORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO) 		return 35;
-			if (num == J9PORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO) 	return 36;
-			if (num == J9PORT_SIG_FLAG_SIGFPE_INT_OVERFLOW) 	return 37;
+		if (num == OMRPORT_SIG_FLAG_SIGQUIT) 	return 3;
+		if (num == OMRPORT_SIG_FLAG_SIGILL) 	return 4;
+		if (num == OMRPORT_SIG_FLAG_SIGTRAP) 	return 5;
+		if (num == OMRPORT_SIG_FLAG_SIGABRT) 	return 6;
+		if (num == OMRPORT_SIG_FLAG_SIGFPE) {
```

Comment:
This is incorrect: `OMRPORT_SIG_FLAG_SIGFPE` is normally paired with another bit, so the next three `if` tests can never succeed.

Please correct the formatting while you're here, the body of each `if` statement should be a block, not on the same line, for example:
```
		if (num == OMRPORT_SIG_FLAG_SIGQUIT) {
			return 3;
		}
```

Reply:
Oops, good catch. Pushed a new commit (and squashed).